client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from 'react';
+import Booking from '../../components/Booking/Booking';
+import { BookingRequest } from '../../interface/BookingRequest';
+import { mockBookingRequest } from '../../mocks/mockBookingRequest';
+
+import { Grid, Paper, Typography, Avatar, Container, Box } from '@material-ui/core';
+import { MuiPickersUtilsProvider, DatePicker } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+
+import useStyles from './useStyles';
+
+const ManageBookings = (): JSX.Element => {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+
+  const history = useHistory();
+  const classes = useStyles();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+
+  const [date, changeDate] = useState<MaterialUiPickersDate>(new Date());
+  const [isSelectedDate, setIsSelectedDate] = useState<boolean>(false);
+
+  const [requests, setRequests] = useState<Array<BookingRequest>>(mockBookingRequest);
+  const nextBooking = requests.reduce((request, closest) =>
+    new Date(request.start.getTime() - new Date().getTime()) < closest.start &&
+    request.start &&
+    request.start > new Date()
+      ? (closest = request)
+      : (closest = closest),
```

Comment:
This seems a bit too complicated🤔, we can probably simplify things a bit:  

All we would want here is to compare `closest.start` and `request.start` to `Date()` and return whichever one is closer (`closest` or `request`). To make sure `closest` is always set, we can provide an initial value (e.g. `requests[0]`) to the reduce call ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce#syntax)) 👍🏼